eval/http.go

```diff
@@ -552,13 +552,39 @@ func deleteHeader(val cty.Value, headerCtx http.Header) {
 }
 
 func toSlice(val interface{}) []string {
+	// as this is called with values from a map returned by seetie.ValueToMap(), val can currently be one of
+	// * nil
+	// * bool
```

Comment:
Should bool values be `[]string`ed, too?
```go
case bool:
	return []string{strconv.FormatBool(v)}
```

Reply:
`eval.toSlice()` is used for
* `set_query_params`
* `add_query_params`
* `set_form_params`
* `add_form_params`
* `set_request_headers`
* `add_request_headers`
* `set_response_headers`
* `add_response_headers`
* `headers`

attributes.

If I add the bool case, a difference would be that in TestFunctions() the response header `x-default-12` resulting from
```
          x-default-12 = default(request.cookies.undef, false)
```
is actually _set_ (with value `"false"`) in contrast to the current expectation (the header _not_ being set).

I don't remember whether setting a value to `false` was an intended means to not actually set a param/header. At least, there would still be setting a value to `null` to not set a param/header.

Reply:
I'll change that in another branch/PR

Reply:
[See #805](https://github.com/coupergateway/couper/pull/805)